apis/sharing/v1alpha1/resourceoffer_types.go

```diff
@@ -35,19 +31,24 @@ type ResourceOfferSpec struct {
 type OfferPhase string
 
 const (
+	// ResourceOfferPending indicates a pending phase, an action is required.
+	ResourceOfferPending OfferPhase = "Pending"
```

Comment:
Would be more explicit in case a manual action is required, with something like WaitingForManualAcceptance, ManualActionRequired or ManualAcceptanceRequired. To me, pending would refer more to an automatic process that still needs to complete (should definitely be kept, but I would distinguish the other case).